sdks/java/nexmark/src/test/java/org/apache/beam/sdk/nexmark/PerfsToBigQueryTest.java

```diff
@@ -92,7 +91,11 @@ public void testSavePerfsToBigQuery() throws IOException, InterruptedException {
     HashMap<NexmarkConfiguration, NexmarkPerf> perfs = new HashMap<>(2);
     perfs.put(nexmarkConfiguration1, nexmarkPerf1);
     perfs.put(nexmarkConfiguration2, nexmarkPerf2);
-    Instant start = Instant.now();
+    // when read using TableRowJsonCoder the timstamp field is boxed into an Integer, cast it to int
+    // to have same types between expected and actual rows. To avoid averflow on int capacity,
+    // set the instant to a fixed date, Beam incubation date :)
+    java.time.Instant start = java.time.Instant.ofEpochSecond(1454284800L);
```

Comment:
1. the overflow was just if we used Instant.now() which will be above int capacity in 2038. Now that the date is fixed, no more overflow problem.
 +1 to your proposed changes, but as I mentioned I will try using only joda time to avoid mixing joda.time and java.time